### PR TITLE
Schedule Changes

### PR DESCRIPTION
**Pull request by javipacheco**
This PR fixes the follow problems:

- Reload Schedule List fixed when add to Favorites
- Favorites not persisted when changing conference locations for the first time
- Added link to twitter account from speaker into the schedule detail
- Added avatar to speakers to the schedule cell
- Added MulltiDex for fix 65k methods

@fedefernandez can you please review? Thanks!

### CODE REVIEW

File: src/main/scala/com/fortysevendeg/android/scaladays/ui/schedule/ScheduleAdapter.scala [hunk 1]
```diff
@@ -29,9 +29,9 @@ import com.fortysevendeg.macroid.extras.TextTweaks._
 import com.fortysevendeg.macroid.extras.ViewGroupTweaks._
 import com.fortysevendeg.macroid.extras.ViewTweaks._
 import macroid.FullDsl._
-import macroid.{ActivityContextWrapper,ContextWrapper}
+import macroid.{Tweak, Ui, ActivityContextWrapper, ContextWrapper}
 
-class ScheduleAdapter(timeZone: String, scheduleItems: Seq[ScheduleItem], listener: RecyclerClickListener)
+case class ScheduleAdapter(timeZone: String, scheduleItems: Seq[ScheduleItem], listener: RecyclerClickListener)
```
Comment (fedefernandez) [inline src/main/scala/com/fortysevendeg/android/scaladays/ui/schedule/ScheduleAdapter.scala]: :+1:

File: src/main/scala/com/fortysevendeg/android/scaladays/ui/scheduledetail/ScheduleDetailActivity.scala [hunk 2]
```diff
@@ -126,6 +131,7 @@ class ScheduleDetailActivity
   private def favoriteClick(eventTitle: String, name: String) = {
     favoriteChanged = true
     val isFavorite = preferenceServices.fetchBooleanPreference(PreferenceRequest[Boolean](name, false)).value
+    isFavoriteEvent = !isFavorite
```
Comment (fedefernandez) [inline src/main/scala/com/fortysevendeg/android/scaladays/ui/scheduledetail/ScheduleDetailActivity.scala]: Can't see usages for this field

Comment (javipacheco) [inline src/main/scala/com/fortysevendeg/android/scaladays/ui/scheduledetail/ScheduleDetailActivity.scala]: I was implementing the ticket and finally I changed the implementation and forget that. I remove it now. Thanks!

**Comment by fedefernandez** [comment]
LGTM!

**Pull request merged**